docs/server/commands.rst

```diff
@@ -581,6 +581,50 @@ and the process will start a watchdog to track any client-side scripts for
 changes. Use this only when developing Pootle.
 
 
+.. _commands#user-management:
+
+Managing users
+--------------
+
+.. django-admin:: merge_user
+
+merge_user
+^^^^^^^^^^
+
+.. versionadded:: 2.7.1
+
+If you have a user with two accounts or need to merge one account into another
+you can use `merge_user`. This will re-assign all submission, units and
```

Comment:
I don't particularly like explanations for a command that refer to the command itself. Same below for the other command.